lib/sprockets/sass/v3/functions.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+module Sprockets
+  module Sass
+    module V3
+      # Module used to inject helpers into SASS engine
+      module Functions
```

Comment:
It'd be nice to maintain compatibility with sprockets-helpers - but that could be done in a separate PR if it's a challenge